src/survey/serializers.py

```diff
@@ -1,9 +1,32 @@
+import json
+
 from rest_framework import serializers
+from rest_framework.fields import SerializerMethodField
+from survey import constants
 from survey.models import Survey
 
 
 class SurveySerializer(serializers.ModelSerializer):
+
+    did_not_meet_reasons = SerializerMethodField("get_did_not_meet_reasons")
+
+    def get_did_not_meet_reasons(self, survey):
+        if survey.did_not_meet_reasons is None:
+            return []
+        return map(
+            lambda x: constants.short_to_long(x),
```

Comment:
if you do a dictionary like i mentioned, you can just return `dict.keys()` or `dict.values()` or something easier like that instead of using convoluted lambda funcs

Reply:
when @alanna-zhou  doesn't like my lambda functions 😢

Reply:
yes good call

Reply:
uhh wait I still need to use a lambda bc i'm converting from keys to values (or the other direction), let me know if you had another solution in mind